docs/usage/python.md

```diff
@@ -57,12 +65,12 @@ some-other-package==1.0.0
 
 ### Sources in `Pipfile`
 
-Renovate detects any custom-configured sources in `Pipfile` and uses them.
+Renovate finds and uses any custom sources in your `Pipfile`.
```

Comment:
I think we should add an example on how to set a custom source in your `Pipfile`.